events/test_eb_api.py

```diff
@@ -0,0 +1,22 @@
+import csv
```

Comment:
This test should be under the tests directory as `tests/eb_riverkeeper_test.py` Here's what you could put in that file:


```python
from os import path
import sys
import unittest
sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
from events.eb_riverkeeper import main
from .utils import schema_test_required, schema_test_all, schema_test_types

class EbRiverkeeperTestCase(unittest.TestCase):
    '''
    Test cases for Eventbrite Riverkeeper events.
    '''

    def setUp(self):
        pass

    def tearDown(self):
        pass

    def test_main(self):
        events = main()
        tests = [schema_test_required, schema_test_all, schema_test_types]
        for test in tests:
            with self.subTest(test = test):
                result = test(events)
                self.assertTrue(result)

if __name__ == '__main__':
    unittest.main()
```

Then, from the root of the repo with your environment activated, you can run your script through this test suite with:
```python
python -m unittest tests/eb_riverkeeper_test.py
```